doc/Learn/Http/HttpOverview.md

```diff
@@ -53,10 +48,10 @@ protected override void OnLaunched(LaunchActivatedEventArgs args)
     var builder = this.CreateBuilder(args)
         .Configure(host => {
             host
-            .UseHttpRefit((context, services) =>
+            .UseHttp((context, services) =>
             {
                 services
-                .AddRefitClient<IShowService>(context,
```

Comment:
This one should stay as AddRefitClient no? It's just the UseHttpKiota and UseHttpRefit that need to change